source/includes/page-templates/update/update.rst

```diff
@@ -0,0 +1,265 @@
+values in |single-or-multiple|.
+
+The {+driver-short+} provides the following methods to update values:
+
+- |sync-method|: Updates one or more fields in |single-or-multiple|.
+- |async-method|: The asynchronous version of |sync-method|.
+
+The following sections describe these methods in more detail.
+
+.. include:: /includes/method-overloads.rst
+
+.. include:: /includes/atlas-sample-data.rst
+
+Methods and Parameters
+----------------------
+
+The |sync-method| and |async-method| methods accept the following parameters:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+
+   * - ``filter``
+     - An instance of the ``FilterDefinition`` class that specifies the |document-or-documents|
+       to update.
+       To learn how to create a query filter, see :ref:`csharp-specify-query`.
+
+       **Data Type:** `FilterDefinition <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.FilterDefinition-1.html>`__
+
+   * - ``update``
+     - An instance of the ``UpdateDefinition`` class. This object specifies the kind of update
+       operation, the fields to update, and the new value for each field. To learn how to
+       create an ``UpdateDefinition`` object, see |fields-link| and |arrays-link|.
+
+       **Data Type:** `UpdateDefinition<TDocument> <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.UpdateDefinition-1.html>`__
+
+   * - ``options``
+     - *Optional.* An instance of the ``UpdateOptions`` class that specifies the
+       configuration for the update operation. The default value is ``null``. For a list
+       of available option, see :ref:`csharp-update-options`.
+
+       **Data Type:** `UpdateOptions <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.UpdateOptions.html>`__
+
+   * - ``cancellationToken``
+     - *Optional.* A token that you can use to cancel the operation.
+
+       **Data type**: ``CancellationToken``
+
+Update Multiple Values
+----------------------
+
+The |sync-method| and |async-method| methods each accept only one
+``UpdateDefinition`` object. The following sections describe how
+to update multiple values in a single method call.
+
+Combined Update Definitions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``Builders.Update.Combine()`` method lets you combine multiple ``UpdateDefinition``
+objects. This method accepts the following parameter:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+
+   * - ``updates``
+     - An array of update definitions to combine.
+
+       **Data Type:** ``UpdateDefinition<TDocument>[]``
+
+The ``Combine()`` method returns a single ``UpdateDefinition`` object that defines
+multiple update operations.
+
+The following code example uses the ``Combine()`` method to combine a
+:manual:`$set </reference/operator/update/set/#mongodb-update-up.-set>` operation and an
+:manual:`$unset </reference/operator/update/unset/#mongodb-update-up.-unset>`
+operation:
```

Comment:
No, good catch. I think my intention was to show that you can combine update definitions of both fields and arrays, but forgot to update the intro text. I also think it works better with set/unset because it matches the following pipeline example.